api-testing/multicloud-api/src/main/java/com/soda/utils/Logger.java

```diff
@@ -0,0 +1,59 @@
+/*
+  Copyright 2020 The SODA Authors.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
+package com.soda.utils;
+
+/**
+ * This class used for logs print.
+ * TODO: Need to create generic method.
+ */
+public class Logger {
+
+    /**
```

Comment:
Why are we using this custom logging.
Java support good library support for logging like log4j
[https://logging.apache.org/log4j/2.x/](url)

Reply:
I am trying to reduce project size. Also it may be problematic when methods are deprecated. So we try to use most of the native code.

Reply:
This is repeated questions. Already discuss with @kumarashit

Reply:
I think using log4j is good option.

Reply:
I have created generic method. Please review

Reply:
I have created generic function as per your previous comment. We can do it using log4j but it will take time again.